packages/core/src/loaders/sequentialRetrieveConfiguration.ts

```diff
@@ -0,0 +1,19 @@
+import type { IRetrieveConfiguration } from '../types';
+
+/**
+ * This simply retrieves the images sequentially as provided.
+ */
+const sequentialRetrieveConfiguration: IRetrieveConfiguration = {
```

Comment:
seems like this file and the interleavedRetrieveConfiguration shoud be moved out of this loaders folder (which are actual loaders), you can put it maybe in src/loaders/progressiveConfigurations?